### ISSUE TITLE
Possible DeadlyReentry issue

### ISSUE THREAD

**Issue by mgsdk** [KSP-CKAN/CKAN-meta#482]
With the recent release of 7.0.0, DeadlyReentry has been upgraded to support KSP 1.0.2. While there are a few PR for the netkan to reflect this, we already have .ckan file for 7.0.0 and 7.0.1. When the netkan metadata updates, I guess the 7.0.0 .ckan file will not be updated.

Currently, it should be visible for users of KSP 0.90, but I am unsure if it supports that version of KSP. Can someone with a 0.90 KSP version check the compatibility between DeadlyReentry 7.0.0 (And perhaps also 7.0.1 in case of a positive result) and report back, as I don't have access to a KSP 0.90 install.

TL;DR: We might have stale, invalid metadata for DeadlyReentry in the form of the version 7.0.0 file. Can someone with a KSP 0.90 install check this.

**Comment by mgsdk** [comment]
Fixed in https://github.com/KSP-CKAN/CKAN-meta/commit/8cbe9d8c0ad99acee8af5323e4faeea43901f176.
